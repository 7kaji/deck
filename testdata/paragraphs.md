# paragraphs

Paragraph1

Paragraph2  
inline line break

Paragraph3
