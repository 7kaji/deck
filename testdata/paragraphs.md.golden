[
  {
    "layout": "",
    "titles": [
      "paragraphs"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "Paragraph1"
              }
            ]
          },
          {
            "fragments": [
              {
                "value": "Paragraph2\n"
              },
              {
                "value": "inline line break"
              }
            ]
          },
          {
            "fragments": [
              {
                "value": "Paragraph3"
              }
            ]
          }
        ]
      }
    ],
    "title_heading_level": 1
  }
]